Comp2023/src/main/java/frc/robot/commands/ElbowMoveToAngle.java

```diff
@@ -0,0 +1,56 @@
+
+// ROBOTBUILDER TYPE: Command.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants.ARMConsts.ElbowAngle;
+import frc.robot.subsystems.Arm;
+
+/**
+ *
+ */
+public class ElbowMoveToAngle extends CommandBase
+{
+  private final Arm  m_elbow;

```

Comment:
The subsystem being passed in is the Arm subsystem--not the elbow (not m_elbow). We're just using the elbow joint from the arm subsystem.

Reply:
zara

Reply:
Marking as resolved on Alex's PR--will be handled on Zara's.